statsmodels/regression/tests/test_lme.py

```diff
@@ -803,6 +803,51 @@ def test_summary_xname_re(self):
 # ------------------------------------------------------------------
 
 
+class TestMixedLMSummaryRegularized:
```

Comment:
are the test methods here exact copies of the unregularized version?

If yes, then you can just subclass TestMixedLMSummary and only define the `setup_class` methods, inheriting the test methods

Reply:
Will do.

Reply:
Done